components/inlinealert/stories/inlinealert.stories.js

```diff
@@ -49,14 +69,28 @@ export default {
 				category: "Advanced",
 			},
 			control: "boolean",
+			if: { arg: "hasLink", truthy: false },
```

Comment:
Are the link and the closable button mutually exclusive? I see both displaying at the same time in the spec. If both can appear, we'll want to make sure both controls are allowed to be on, and both being used at the same time are included in the tests file.

<img width="122" alt="Screenshot 2025-04-25 at 4 41 20 PM" src="https://github.com/user-attachments/assets/cca0a1bd-3d8c-4542-a73f-26d9818bb77b" />